### PR TITLE
feat: handle multiple VCALENDAR components in a single ICS file

### LINKED ISSUE #22: multiple calendars in one file [awareness|edge case]
.ics files may contain multiple vcalendars. `icalendar` allows parsing them with `multiple=True`.
This can be considered.

Test:

- [ ] merge multiple files with multiple calendars

### PR DESCRIPTION
**Pull request by SashankBhamidi**
Closes #22

`.ics` files can contain multiple concatenated `VCALENDAR` blocks. Previously this caused a `ValueError` when parsing. This adds `calendars_from_ical()` using `Calendar.from_ical(data, multiple=True)` as the single parsing entry point across the CLI and test helpers.

### CODE REVIEW

**Review by abe-101 — COMMENTED** [review]
Change looks good to me; Would like to get @niccokunzmann review for approval

File: tests/conftest.py [hunk 1]
```diff
@@ -51,7 +51,8 @@ class ICSCalendars:
 
     def get_calendar(self, content):
         """Return the calendar given the content."""
-        return icalendar.Calendar.from_ical(content)
+        cals = calendars_from_ical(content)
```
Comment (niccokunzmann) [inline tests/conftest.py]: It would be good to keep a uniform interface that for testing also allows accessing all calendars in the calendar stream.

```suggestion
        cals = calendars_from_ical(content)
        cal = cals[0]
        cal.stream = cals
        return cal
```

**Review by niccokunzmann — CHANGES_REQUESTED** [review]
Hm. It seems we already have a fixture to access the calendars.
Also, keeping a uniform interface is quite nice.

**Comment by SashankBhamidi** [comment]
@niccokunzmann I deferred module-scoped dict fixture, adds complexity without performance benefit for current test scope.

**Review by niccokunzmann — CHANGES_REQUESTED** [review]
Generally, it is good to use the existing fixtures. You could even remove `two_vcalendars_cals` to make the tests slimmer.

I think this should work somehow like this now.

**Review by niccokunzmann — DISMISSED** [review]
Cool. Nice that that works!
